common/scaffolding/typographie.html

```diff
@@ -0,0 +1,39 @@
+---
+{
+	"title": "Typographie",
+	"language": "fr",
+	"altLangPage": "typography.html",
+	"breadcrumbs": [
+		{ "title": "GCWeb home", "link": "https://wet-boew.github.io/GCWeb/index-en.html" }
+	],
+	"dateModified": "2023-08-24"
+}
+---
+
+<h1>Titres</h1>
+<h1>Il s'agit d'un élément de titre h1</h1>
+<h2>Il s'agit d'un élément de titre h2</h2>
+<h3>Il s'agit d'un élément de titre h3</h3>
+<h4>Il s'agit d'un élément de titre h4</h4>
+<h5>Il s'agit d'un élément de titre h5</h5>
+<h6>Il s'agit d'un élément de titre h6</h6>
+<pre><code>
+&lt;h1&gt;Il s'agit d'un élément de titre h1&lt;/h1&gt;
+&lt;h2&gt;Il s'agit d'un élément de titre h2&lt;/h2&gt;
+&lt;h3&gt;Il s'agit d'un élément de titre h3&lt;/h3&gt;
+&lt;h4&gt;Il s'agit d'un élément de titre h4&lt;/h4&gt;
+&lt;h5&gt;Il s'agit d'un élément de titre h5&lt;/h5&gt;
+&lt;h6&gt;Il s'agit d'un élément de titre h6&lt;/h6&gt;
+</code></pre>
+
+<h1>Paragraphe</h1>
+<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+<pre><code>
+&lt;p&gt;Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.&lt;/p&gt;
+</code></pre>
+
+<h1>Largeur de contenu limitée</h1>
+<p>Pour un exemple de contenu de largeur limitée, visitez le lien suivant: <a href="https://wet-boew.github.io/GCWeb/templates/content-limit-en.html">Page de contenu - Largeur de contenu limitée</a></p>
+
+<h1>Sémantique au niveau du texte</h1>
```

Comment:
Except for a few exception, a page should only contain one heading level 1

Reply:
Fixed and addressed in commit "Updated headings and added paragraph".